cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sources/StreamBatchSource.java

```diff
@@ -98,6 +98,18 @@ public StreamBatchSource(StreamBatchConfig streamBatchConfig) {
 
   @Override
   public void configurePipeline(ETLStage stageConfig, PipelineConfigurer pipelineConfigurer) {
+    pipelineConfigurer.addStream(new Stream(streamBatchConfig.name));
```

Comment:
still want to check the schema here. If it's an invalid schema we want to fail early at adapter creation